### PR TITLE
New Song: Abstract Nonsense

### PR DESCRIPTION
**Pull request by Abc2557**
~~Whoop, I forgot how to make a pull request, so now I'm just praying I did it right~~

The song is Abstract Nonsense by Neru.
The song is originally in Japanese, so I kinda just used lyrics from a somewhat popular English translation.

Note: I may end up switching the lyrics out for another translation

### CODE REVIEW

File: Monika After Story/game/script-songs.rpy [hunk 1]
```diff
@@ -1355,3 +1355,37 @@ label mas_monika_plays_or(skip_leadin=False):
...
+            category=[store.mas_songs.TYPE_SHORT],
+            prompt="Abstract Nonsense",
+            random=True,
+            aff_range=(mas_aff.NORMAL,None)
+        ),
+        code="SNG"
+    )
+
+label mas_song_abstract_nonsense:
+    m 2dfd "{i}~This is useless,{/i}{w=0.5}"
```
Comment (LooseEthics) [inline Monika After Story/game/script-songs.rpy:1373]:
```suggestion
    m 2dfd "{i}~This is useless,{/i}{w=0.5}{nw}"
```

File: Monika After Story/game/script-songs.rpy [hunk 2]
```diff
@@ -1355,3 +1355,37 @@ label mas_monika_plays_or(skip_leadin=False):
...
+    extend 2rkd "{i} I want it to stop,{/i}{w=0.5}"
```
Comment (LooseEthics) [inline Monika After Story/game/script-songs.rpy:1374]:
```suggestion
    extend 2rkd "{i} I want it to stop,{/i}{w=0.5}{nw}"
```

File: Monika After Story/game/script-songs.rpy [hunk 3]
```diff
@@ -1355,3 +1355,37 @@ label mas_monika_plays_or(skip_leadin=False):
...
+    extend 6rkc "{i} I put my face out the window~{/i}"
+    m 2dfd "{i}~This is boring,{/i}{w=0.5}"
```
Comment (LooseEthics) [inline Monika After Story/game/script-songs.rpy:1376]:
```suggestion
    m 2dfd "{i}~This is boring,{/i}{w=0.5}{nw}"
```

File: Monika After Story/game/script-songs.rpy [hunk 4]
```diff
@@ -1355,3 +1355,37 @@ label mas_monika_plays_or(skip_leadin=False):
...
+    extend 2rkd "{i} I'm tired of it,{/i}{w=0.5}"
```
Comment (LooseEthics) [inline Monika After Story/game/script-songs.rpy:1377]:
```suggestion
    extend 2rkd "{i} I'm tired of it,{/i}{w=0.5}{nw}"
```

File: Monika After Story/game/script-songs.rpy [hunk 5]
```diff
@@ -1355,3 +1355,37 @@ label mas_monika_plays_or(skip_leadin=False):
...
+    extend 6rkc "{i} but I don't have the courage to die~{/i}"
+    m 2dfd "{i}~This is useless,{/i}{w=0.5}"
```
Comment (LooseEthics) [inline Monika After Story/game/script-songs.rpy:1379]:
```suggestion
    m 2dfd "{i}~This is useless,{/i}{w=0.5}{nw}"
```

File: Monika After Story/game/script-songs.rpy [hunk 6]
```diff
@@ -1355,3 +1355,37 @@ label mas_monika_plays_or(skip_leadin=False):
...
+    extend 2rkd "{i} I want it to stop,{/i}{w=0.5}{nw}"
+    extend 5rkc "{i} I'm injecting my wrist~{/i}"
+    m 2dfd "{i}~This is boring,{/i}{w=0.5}"
```
Comment (LooseEthics) [inline Monika After Story/game/script-songs.rpy:1382]:
```suggestion
    m 2dfd "{i}~This is boring,{/i}{w=0.5}{nw}"
```

File: Monika After Story/game/script-songs.rpy [hunk 7]
```diff
@@ -1355,3 +1355,37 @@ label mas_monika_plays_or(skip_leadin=False):
...
+    extend 2rkd "{i} I'm tired of it,{/i}{w=0.5}"
```
Comment (LooseEthics) [inline Monika After Story/game/script-songs.rpy:1383]:
```suggestion
    extend 2rkd "{i} I'm tired of it,{/i}{w=0.5}{nw}"
```

File: Monika After Story/game/script-songs.rpy [hunk 8]
```diff
@@ -1355,3 +1355,37 @@ label mas_monika_plays_or(skip_leadin=False):
...
+    extend 1tsc "{i} I eat the cake{/i}{w=0.5}"
```
Comment (LooseEthics) [inline Monika After Story/game/script-songs.rpy:1384]:
```suggestion
    extend 1tsc "{i} I eat the cake{/i}{w=0.5}{nw}"
```

File: Monika After Story/game/script-songs.rpy [hunk 9]
```diff
@@ -1355,3 +1355,37 @@ label mas_monika_plays_or(skip_leadin=False):
...
+    extend 6dsd "{i} and take a deep breath~{/i}"
+    m 1ekd "You know, [player], I really feel bad for the main character of this song..."
+    m 3wkd "And one of the worst parts is that a lot of people really relate to how she feels."
+    m 6dkd "...It really breaks my heart."
+    m 1ekd "Please remember that I'll always be here for you and that you're never really alone."
+    m 1dkb "I really do love you."
+    return "derandom" "love"
```
Comment (LooseEthics) [inline Monika After Story/game/script-songs.rpy:1391]:
```suggestion
    return "derandom|love"
```
For the sake of code consistency. Not sure if return with 2 arguments is actually wrong.

**Comment by jmwall24** [comment]
I'm not really crazy about the message this song sends, even with her closing remarks. It worries me a bit that it may have a negative effect on someone who is already depressed. Also trying not to add too many vocaloids in here, as she's already said she doesn't really listen to them much, and we have a few already plus your other PR.

**Pull request closed without merging**